packages/snaps-sdk/src/jsx/components/form/Field.ts

```diff
@@ -20,7 +20,8 @@ export type FieldProps = {
     | DropdownElement
     | FileInputElement
     | InputElement
-    | CheckboxElement;
+    | CheckboxElement
+    | null;
```

Comment:
I don't think `Field` should accept `null`.

Reply:
If Field don't accept `null` there is no way of validating the Field in the simulator since the struct will always fail when you try to place an empty `Field`.

Reply:
Ah, right. @FrederikBolding do you have any ideas

Reply:
Hmm, could we add an edge-case to the validation in the simulator instead?

I agree with Maarten that `Field` shouldn't accept `null`

Reply:
Updated :)